docs/troubleshooting/common-issues.md

```diff
@@ -101,6 +101,24 @@ $ pex streamlit foo bar -o mypexbinary.pex
   $ ./mypexbinary.pex -m streamlit hello
   ```
 
+## Streamlit hangs when deployed as a web app
+
+If you're Streamlit app hangs when it's deployed as a web app (EC2, Heroku, AppEngine, CloudRun, etc.), you can try disabling CORS.
```

Comment:
We should also say that if you host with Streamlit For Teams, this problem gets solved automatically. Then point people to https://www.streamlit.io/for-teams

Reply:
Can you provide additional details for why this solves the problem? Will this service cost anything?

Reply:
I just went and updated the docs directly in your branch :)

Will merge now, but feel free to open new PRs if you feel like my edits need a pass.